sagemathinc/cocalc
# january 2019 npm update

# Description

This mostly does critical updates (`npm audit`)

# Testing Steps
1. besides travis-ci, this also needs proper staging and manual checks. but overall, I think this isn't dangerous at all.
1. noteworth is that `less` (for whatever we use it, actually) is updated, primus, and engine.io
2. if you review this in your own project, make sure to rebuild primus, too
1. there are also project updates!

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
